frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { useState, useEffect } from "react";
```

Comment:
To match the other comment, you can import Fragment from react directly
```suggestion
import React, { Fragment, useState, useEffect } from "react";
```